build-plugin/debian/copyright

```diff
@@ -0,0 +1,29 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: signalfx-collectd-plugin
+Source: <url://example.com>
+
+Files: *
+Copyright: 2015 SignalFx, Inc. <support+deb@signalfx.com>
+License: GPL-3.0+
```

Comment:
I don't think we are GPL-3.0+ lets talk with jack to figure out what we must be..